src/openassetio-python/package/openassetio/pluginSystem/PythonPluginSystemManagerImplementationFactory.py

```diff
@@ -37,48 +37,97 @@ class PythonPluginSystemManagerImplementationFactory(ManagerImplementationFactor
     derived plugins. Not usually used directly by a @ref host, which
     instead uses the @fqref{hostApi.ManagerFactory} "ManagerFactory".
 
+    The factory loads plugins from exposed `openassetio.manager_plugin`
+    entry points, as well as additional plugins found under paths
+    specified in the `OPENASSETIO_PLUGIN_PATH` env var.
+
     @envvar **OPENASSETIO_PLUGIN_PATH** *str* A PATH-style list of
-    directories to search for @ref openassetio.pluginSystem.PythonPluginSystemManagerPlugin
-    based plugins. It uses the platform-native delimiter.  Searched left
-    to right.
+    directories to search for @ref
+    openassetio.pluginSystem.PythonPluginSystemManagerPlugin based
+    plugins. It uses the platform-native delimiter.  Searched left to
+    right. Plugins found here will take precedence over those discovered
+    through package entry points. Note: this search path is entirely
+    independent of **PYTHONPATH** and doesn't not need to be a subset of
```

Comment:
[Consider] Oddly phrased double negative, `dosen't not need`, consider simplifying.

Reply:
Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/768/commits/21dd2256bfbb9bc4e7e54831516cf9ab448422b7)